superset/databases/dao.py

```diff
@@ -124,3 +125,13 @@ def get_related_objects(cls, database_id: int) -> Dict[str, Any]:
         return dict(
             charts=charts, dashboards=dashboards, sqllab_tab_states=sqllab_tab_states
         )
+
+    @classmethod
+    def get_ssh_tunnel(cls, database_id: int) -> Dict[str, Any]:
```

Comment:
also can we write a small unit test for this here's an example

https://github.com/apache/superset/blob/89020b586687ecf84eef6d7337b6fea767162c9c/tests/unit_tests/datasets/dao/dao_tests.py#L63